warehouse/config.py

```diff
@@ -159,6 +159,7 @@ def configure(settings=None):
     maybe_set(settings, "aws.region", "AWS_REGION")
     maybe_set(settings, "gcloud.credentials", "GCLOUD_CREDENTIALS")
     maybe_set(settings, "gcloud.project", "GCLOUD_PROJECT")
+    maybe_set(settings, "warehouse.distribution_table", "WAREHOUSE_DISTRIBUTION_TABLE")
```

Comment:
I'd prefer `warehouse.release_files_table`/`WAREHOUSE_RELEASE_FILES_TABLE`, again to keep terms consistent among DB models. I don't recall that we use the word "distribution" anywhere in the codebase yet...